engine/src/main/java/org/destinationsol/menu/MainMenuScreen.java

```diff
@@ -29,93 +28,82 @@
+                                                                                if (!isMobile) {
+                                                                                    solApplication.getOptions().save();
+                                                                                }
+                                                                                Gdx.app.exit();
+                                                                            }));
+
+        UiTextButton creditsButton = new UiTextButton().setDisplayName("Credits")
+                                                       .setTriggerKey(Input.Keys.C)
+                                                       .enableSound()
+                                                       .setOnReleaseAction(() -> inputManager.changeScreen(screens.creditsScreen));
+
+        rootUiElement = new UiRelativeLayout().addElement(buttonList, UI_POSITION_BOTTOM, 0, -buttonList.getHeight()/2 - BUTTON_PADDING)
+                                              .addElement(creditsButton, UI_POSITION_BOTTOM_RIGHT, -BUTTON_WIDTH/2 - BUTTON_PADDING, -BUTTON_HEIGHT/2 - BUTTON_PADDING)
+                                              .finalizeChanges();
 
         backgroundTexture = Assets.getAtlasRegion("engine:mainMenuBg", Texture.TextureFilter.Linear);
         logoTexture = Assets.getAtlasRegion("engine:mainMenuLogo", Texture.TextureFilter.Linear);
     }
 
     @Override
     public void updateCustom(SolApplication solApplication, SolInputManager.InputPointer[] inputPointers, boolean clickedOutside) {
-        tutorialControl.setEnabled(solApplication.getOptions().controlType != GameOptions.ControlType.CONTROLLER);
-
-        if (tutorialControl.isJustOff()) {
-            solApplication.play(true, "Imperial Small", true);
-            return;
-        }
-
-        SolInputManager inputManager = solApplication.getInputManager();
-        MenuScreens screens = solApplication.getMenuScreens();
-
-        if (newGameControl.isJustOff()) {
-            inputManager.setScreen(solApplication, screens.newGameScreen);
-            return;
-        }
-
-        if (optionsControl.isJustOff()) {
-            inputManager.setScreen(solApplication, screens.optionsScreen);
-            return;
-        }
-
-        if (exitControl.isJustOff()) {
-            // Save the settings on exit, but not on mobile as settings don't exist there.
-            if (!isMobile) {
-                solApplication.getOptions().save();
-            }
-            Gdx.app.exit();
-            return;
-        }
-
-        if (creditsControl.isJustOff()) {
-            inputManager.setScreen(solApplication, screens.creditsScreen);
-        }
+        // TODO: Disabled for now, since I can't figure out why controller players shouldn't be able to play the tutorial.
```

Comment:
Aren't there somewhere hardcoded text refering to button, that won't be applicable to controller?